crates/ruff_python_formatter/src/statement/suite.rs

```diff
@@ -32,23 +32,59 @@ pub enum SuiteKind {
     Class,
 
     /// Statements in any other body (e.g., `if` or `while`).
-    #[default]
-    Other,
+    Other {
```

Comment:
It's not just statements, e.g. it's also includes `case` in match statements. Maybe `WithSuite`?

Reply:
I find last suite in statement describes it well; the rule is that we insert empty lines except when the statement ends directly after it, because then it's the task of whatever contains the suite to insert empty lines. A different name would be `ends_at_end_of_statement`. For match-case, we capture whether the end of the (case-)suite ends at the end of the match-statement.